### ISSUE TITLE
Cannot install plugin in Blender 2.76b

### ISSUE THREAD

**Issue by Warren93** [der-On/XPlane2Blender#380]
I have tried importing the 3.4 release, but Blender throws errors when I try to enable it.
First it gives me this:

Modules Installed from '/home/Warren/Downloads/io_xplane2blender_3_4_0_rc-2-3_20180515160711.zip' into '/home/Warren/.config/blender/2.76/scripts/addons' ()
Traceback (most recent call last):
  File "/usr/share/blender/2.76/scripts/modules/addon_utils.py", line 324, in enable
    mod = __import__(module_name)
  File "/home/Warren/.config/blender/2.76/scripts/addons/io_xplane2blender/__init__.py", line 47, in <module>
    from . import xplane_ui
  File "/home/Warren/.config/blender/2.76/scripts/addons/io_xplane2blender/xplane_ui.py", line 5, in <module>
    from .xplane_ops import *
  File "/home/Warren/.config/blender/2.76/scripts/addons/io_xplane2blender/xplane_ops.py", line 7, in <module>
    from .xplane_ops_dev import *
  File "/home/Warren/.config/blender/2.76/scripts/addons/io_xplane2blender/xplane_ops_dev.py", line 8, in <module>
    from io_xplane2blender.xplane_types import xplane_lights_txt_parser
  File "/home/Warren/.config/blender/2.76/scripts/addons/io_xplane2blender/xplane_types/__init__.py", line 15, in <module>
    from .xplane_mesh import XPlaneMesh
  File "/home/Warren/.config/blender/2.76/scripts/addons/io_xplane2blender/xplane_types/xplane_mesh.py", line 390
    o += ''.join([s_idx10 % (*self.indices[i:i+10],) for i in range(0,partition_point-1,10)])
SyntaxError: can use starred expression only as assignment target

...and then if I try again it gives me this:

Traceback (most recent call last):
  File "/usr/share/blender/2.76/scripts/modules/addon_utils.py", line 324, in enable
    mod = __import__(module_name)
  File "/home/Warren/.config/blender/2.76/scripts/addons/io_xplane2blender/__init__.py", line 47, in <module>
    from . import xplane_ui
  File "/home/Warren/.config/blender/2.76/scripts/addons/io_xplane2blender/xplane_ui.py", line 5, in <module>
    from .xplane_ops import *
  File "/home/Warren/.config/blender/2.76/scripts/addons/io_xplane2blender/xplane_ops.py", line 7, in <module>
    from .xplane_ops_dev import *
  File "/home/Warren/.config/blender/2.76/scripts/addons/io_xplane2blender/xplane_ops_dev.py", line 7, in <module>
    from io_xplane2blender import xplane_helpers
ImportError: cannot import name 'xplane_helpers'

I am using Blender 2.76b with Python 3.4.6. I'm hoping I don't have to install any later versions of either since I don't have root access on the machine I'm currently using. Any suggestions?

**Comment by tngreene** [comment]
I don't normally recommend doing this because it makes debugging future problems much harder, but, it looks like xplane_mesh.py was only changed between v3.3.12 and v3.4 with the performance optimization fix. Unfortunately in that file I had to use some fancy syntax tricks to squeeze as much performance as possible, and I didn't test it across all versions.

If you can't upgrade your Blender version you could replace the xplane_mesh.py file from v3.4 with one from v3.3.12. Again THIS IS HIGHLY UNORTHODOX, but I think it could work in this instance. The file was last edited 4 months ago by me, then 2 years ago by Onder. No API changes either. Tell me how it works out!

**Comment by tngreene** [comment]
How did it go? Did this hack work or were my instructions too hard to follow?

**Comment by Warren93** [comment]
Sorry for the late reply. I got someone to install the latest Blender on my machine, and the plugin installed and enabled without issue (although I haven't had a chance to try doing anything with it yet). I also tried the solution you mentioned (putting the xplane_mesh.py from 3.3.12 in the 3.4 release) and got the same as stack trace as the second thing I pasted above.

Anyway, thanks for your help, I appreciate it. I expect I'll come back here armed with new questions when I start actually using the plugin.

**Comment by Warren93** [comment]
Whoops, didn't mean to close (problem still technically exists). hit the wrong button.

**Comment by tngreene** [comment]
Closing this because, aside from the hack that didn't work, Blender 2.76b was released almost 3 years ago and we're not supporting it anymore. By now administrators and Distro managers should update their packages and give their users more recent. Its not like this is used in a hospital where they still use Windows XP (terrifying) because of overly rigid regulations and hardware that can't get updated.

If it turns out that a lot of people are stuck with this, we'll make an exception, but so far you've been the only one. I'm super glad you were able to get some help with this! Thanks for reporting your issue!
